src/main/java/com/appleaster/task/Event.java

```diff
@@ -0,0 +1,30 @@
+package com.appleaster.task;
+import java.time.LocalDateTime;
+import java.time.format.DateTimeFormatter;
+
+public class Event extends Task {
+    protected LocalDateTime from;
+    protected LocalDateTime to;
+    private static final DateTimeFormatter INPUT_FORMAT = DateTimeFormatter.ofPattern("yyyy-MM-dd HHmm");
+    private static final DateTimeFormatter OUTPUT_FORMAT = DateTimeFormatter.ofPattern("MMM d yyyy, hh:mm a");
+
+    public Event(String description, String from, String to) {
+        super(description, TaskType.EVENT);
+        this.from = LocalDateTime.parse(from, INPUT_FORMAT);
+        this.to = LocalDateTime.parse(to, INPUT_FORMAT);
+    }
+
+    public LocalDateTime getFrom() {
+        return from;
+    }
+
+    public LocalDateTime getTo() {
+        return to;
+    }
+
+    @Override
+    public String toString() {
```

Comment:
Utilise the OOP concept of inheritance by abstracting out the common information such as description to be returned by the superclass/parent Task.java instead before Overriding and adding more specific information to reduce the need to modify multiple files for a change.